tests/compliance_v2_test.go

```diff
@@ -0,0 +1,117 @@
+package tests
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	v2 "github.com/stackrox/rox/generated/api/v2"
+	"github.com/stackrox/rox/pkg/retry"
+	"github.com/stackrox/rox/pkg/testutils/centralgrpc"
+	"github.com/stackrox/rox/pkg/uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestComplianceGetProfiles(t *testing.T) {
```

Comment:
You may not need this test. I think we should combine these tests in the ones @rhmdnd started to prove that everything installed correctly.  I would recommend moving these tests to `tests/compliance_operator_v2_test.go`.  @rhmdnd thoughts?